argon2/src/block.rs

```diff
@@ -66,8 +66,10 @@ impl Block {
         unsafe { &mut *(self.0.as_mut_ptr() as *mut [u8; Self::SIZE]) }
     }
 
+    /// NOTE: do not call this directly. It should only be called via
+    /// `Argon2::compress`.
     #[inline(always)]
```

Comment:
I suspect this would be fine as `#[inline]` but haven't benchmarked yet

Reply:
AFAIK `#[inline]` is effectively useless for crate-private functions. You could remove the attribute completely  to check whether compiler decides to inline it by itself. But to be on the safe side and to be more robust in the face of future compiler changes, I think it's fine to use `#[inline(always)]`.